Dockerfile

```diff
@@ -39,37 +39,41 @@ RUN mkdir /opt/kimai && \
 #fpm alpine php extension base
 FROM php:7.3.10-fpm-alpine3.10 AS fpm-alpine-php-ext-base
 RUN apk add --no-cache \
-    # gd
```

Comment:
have you sort the tools alphabetical or what happens here?

Reply:
I sorted the groups alphabetically.

Reply:
I guess they should really all be alphabetical